contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -4290,15 +4290,13 @@ makeInsertBulkStatement(TSqlParser::Dml_statementContext *ctx)
 		/* create a list of columns to insert into */
 		if (!column_list.empty())
 		{
-			for (size_t i = 0; i < column_list.size() - 1; i++)
+			for (size_t i = 0; i < column_list.size(); i++)
 			{
-				if (column_list[i]->simple_column_name())
```

Comment:
1. Based on the grammar, column_list[i] will always have simple_column_name
```
insert_bulk_column_definition
    : simple_column_name (data_type system_versioning_column? | AS expression PERSISTED? ) ( special_column_option | (COLLATE (id | DEFAULT)) | null_notnull )*
      ( column_constraint? IDENTITY (LR_BRACKET sign? seed=DECIMAL COMMA sign? increment=DECIMAL RR_BRACKET)? )? for_replication? ROWGUIDCOL?
      column_constraint* column_inline_index?
    ;
```
column_list NULL check has been taken care.
We only need to check if column_refs is NOT NULL

2. We downcase the column names here, since later we directly use the same list to match with the the table's columns (which by default will be lowercase in PG)

Reply:
just make sure that we do not need original name in some other context otherwise are loosing it from here itself